recipe/meta.yaml

```diff
@@ -58,6 +58,6 @@ extra:
   recipe-maintainers:
     - mwcraig
     - bsipocz
```

Comment:
I don't really have time anymore to shepherd conda-forge packages besides hitting the merge button quickly on passing PRs. So I suppose there is no use of keeping this around, unless you want that quick merge ;)

```suggestion
```

Reply:
This suggestion makes sense since this is a STScI package.

Reply:
Thanks!